Automattic/sensei
# Integrate update endpoint

### Changes proposed in this Pull Request

* Intigrates the extensions page with the REST API.
* Only update is implemented with this PR. Install and possibly Renew will be handled by other PRs

### Testing instructions

* You can simulate a plugin being outdated by modifying the version in the main plugin file.
* When testing plugins that are hosted in WC.com, WooCommerce save the plugin information in a transient, which means that you might need to do a couple of refreshes for the update to work. Otherwise you will get an error saying that the plugin is already at the latest version.
* After modifying the plugin files, navigate to the extension page and either click the Update button in the notification, or in one of the plugins.
* Observe that the plugins are updated.
